rust-runtime/smithy-client/src/retry.rs

```diff
@@ -380,7 +398,7 @@ mod test {
     #[test]
     fn backoff_timing() {
         let mut conf = test_config();
-        conf.max_retries = 5;
+        conf.max_attempts = 5;
```

Comment:
tbh, I'm having a hard time wrapping my brain around this...I wonder if we should keep the controller as operating in `# of retry` land and just convert it when it comes into this code